### ISSUE TITLE
Multiple warnings during build

### ISSUE THREAD

**Issue by stuarthayhurst** [ubuntu/yaru#2621]
Hi, there wasn't really a fitting template, so I'll try be as helpful as I can :)
When compiling and installing yaru via the instructions in `CONTRIBUTING.md`, I get several warnings during build. I don't know if they're related, I'll file separate bug reports if needs be.
The first issue is when running `meson build`, i get the following:
```
WARNING: Output file "gtksourceview/src/Yaru-dark.xml" for configure_file() at gtksourceview/src/meson.build:13 overwrites configure_file() output at gtksourceview/src/meson.build:13
Configuring Yaru-dark.xml using configuration
WARNING: Output file "gtksourceview/src/Yaru-dark.xml" for configure_file() at gtksourceview/src/meson.build:13 overwrites configure_file() output at gtksourceview/src/meson.build:13
Configuring Yaru-dark.xml using configuration
Configuring Yaru-light.xml using configuration
WARNING: Output file "gtksourceview/src/Yaru-light.xml" for configure_file() at gtksourceview/src/meson.build:13 overwrites configure_file() output at gtksourceview/src/meson.build:13
Configuring Yaru-light.xml using configuration
WARNING: Output file "gtksourceview/src/Yaru-light.xml" for configure_file() at gtksourceview/src/meson.build:13 overwrites configure_file() output at gtksourceview/src/meson.build:13
```

The second and third issues are with `sudo ninja install`. I get deprecation warnings. The first of these 2 issues is between lines 53 and 97. The other issue is present throughout almost the entire log, and is warning about symlink copying behaviour changing in future. (Attached as `ninja.log`)

`git describe` gives me `20.10.6.1-127-ga78013dd`, the latest commit as far as I can tell

[meson.log](https://github.com/ubuntu/yaru/files/5939529/meson.log)
[ninja.log](https://github.com/ubuntu/yaru/files/5939530/ninja.log)

**Comment by Feichtmeier** [comment]
@flexiondotorg could you have a look at the gtksourceview part?

**Comment by Muqtxdir** [comment]
These have fixed by @3v1n0 in PR: #3416
